### DISCUSSION TITLE
Can the .bin file be modified

### DISCUSSION THREAD

**Discussion by JoeInDowney** [Jyers/Marlin#882] [Q&A] [answered]
I loaded the E3V2default bin file into VS code and garbled text was displayed. Is there a way to view it and add support for the EZABL? I have the lines to be added and commentated. Thank you.

![Capture](https://user-images.githubusercontent.com/80736627/119232745-36819c00-badb-11eb-84c9-67ae99cfcf2c.PNG)

**Comment by smooreace** [comment, marked as answer]
You cant edit a compiled file... thats what the source code is for

**Comment by Cal-LF** [comment]
Compiling instructions may help. 
[Compiling Steps.txt](https://github.com/Jyers/Marlin/files/6526787/Compiling.Steps.txt)

**Comment by smooreace** [comment]
There are dozens of videos out there. All too easy to find.

**Reply by JoeInDowney** [reply]
Thank you.
